### PR TITLE
Mining base remap

### PR DESCRIPTION
**Pull request by niceping**
It will not end well
pictures are kind of outdated a bit

bandicam 2018-10-13 19-16-42-448
bandicam 2018-10-13 19-10-57-613

:cl: mining update
tweak: Reworked mining station. It won't end well for sure.
/:cl:

### CODE REVIEW

**Comment by niceping** [comment]
won't do _maps/sunset/ because i'm just lazy so will leave it just there for someone else to do that

**Review by ghost — CHANGES_REQUESTED** [review]
A way to do it that guarantees zero merge conflicts is this:

Make new copy of Lavaland.dmm (_maps/map_files/Mining/SunsetLavaland.dmm), put your changes in IT

Change `LoadGroup(FailedZs, "Lavaland", "map_files/Mining", "Lavaland.dmm", default_traits = ZTRAITS_LAVALAND)` to say `LoadGroup(FailedZs, "Lavaland", "map_files/Mining", "SunsetLavaland.dmm", default_traits = ZTRAITS_LAVALAND) // sunset -- use our lavaland instead`

https://github.com/SunsetStation/SunsetStation/blob/master/code/controllers/subsystem/mapping.dm#L240

**Comment by niceping** [comment]
>blob
what

**Comment by niceping** [comment]
also change basemap too?

**Comment by ghost** [comment]
yup

**Comment by niceping** [comment]
2 commits are not appearing hmms

**Review by ghost — APPROVED** [review]
LGTM

**Review by alexkar598 — CHANGES_REQUESTED** [review]
Power runs out really quickly

**Comment by niceping** [comment]
@alexkar598 It's probably because old mining SMES had more power, maybe even full. I dunno how 3 APCs can drain SMES that though isn't fully charged, but you can just adjust power output. Also, PACMANs exists for reasons, crowbar in miners box too

**Comment by niceping** [comment]
yeah i found the problem, SMES charge is 50000, while old mining had much more. so yeah, just go rush for plasma cutters as usual, and get some plasma for pacmans

**Review by ghost — CHANGES_REQUESTED** [review]
fix the SMES problem. "just get a pacman" isn't an excuse

**Comment by niceping** [comment]
@steamp0rt @alexkar598 done

**Comment by alexkar598** [comment]
mapdiffbot you there?

File: _maps/map_files/Mining/Lavaland.dmm [hunk 1]
```diff
@@ -3708,6 +3708,7 @@
 /area/mine/production)
 "rP" = (
 /obj/machinery/atmospherics/pipe/layer_manifold{
+	icon_state = "manifoldlayer";
```
Comment (alexkar598) [inline _maps/map_files/Mining/Lavaland.dmm]: unneeded change,please remove
```suggestion

```

Comment (niceping) [reply _maps/map_files/Mining/Lavaland.dmm]: @alexkar598 Dunno how the fuck that happened since i didnt touched normal mining but ok

**Comment by niceping** [comment]
todo:
bump smes charge to 33% or more because people (still) complain about charge
fix medkit
medbay airlocks
replace showcase microwave with normal one because thats just cruel
more suits (somewhere)

**Comment by niceping** [comment]
@steamp0rt @alexkar598 beep boop

**Comment by ghost** [comment]
Can you please pressurize the walkway? Just maybe have it be reinforced glass.

**Comment by niceping** [comment]
I can but isnt open area seems like cool idea? anyways ok i will do it

**Comment by niceping** [comment]
doesn't compiles?

FUCK

**Comment by niceping** [comment]
help travis griff me

**Comment by miniusAreas** [comment]
Its a great idea to remap mining, but it feels more bare and naked than our current one. Also is missing a fuckload of the cargotile decals

**Comment by niceping** [comment]
screenshots are outdated, in final version there is much more cargotiles

**Comment by niceping** [comment]
(possibly) going to close this in favor of (possible) gulag remap

**Pull request closed without merging**